src/Carbon/CarbonInterval.php

```diff
@@ -1825,15 +1823,15 @@ public function __toString()
     {
         $format = $this->localToStringFormat;
 
-        if ($format) {
-            if ($format instanceof Closure) {
-                return $format($this);
-            }
+        if ($format === null) {
```

Comment:
This is maybe not a case we need to support, but this will change the behavior when `$format` is an empty string (e.g. for `->settings(['toStringFormat' => ''])`)

For now I would stick to:
```suggestion
        if (!$format) {
```
So the behavior is the same and consistent with the date classes.